pkg/virtctl/expose/expose.go

```diff
@@ -125,6 +128,16 @@ func (o *Command) RunE(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("unknown service type: %s", strServiceType)
 	}
 
+	// convert from string to IPFamily type enum
+	switch strIPFamily {
+	case "ipv4":
```

Comment:
what about using lowercase version of v1.IPv4Protocol ? and same for ipv6

Reply:
Not sure I understand you; I'l l just lowercase the variable, which seems fairer - the user's intentions if 'ipV4'  is type are clear.

Reply:
I mean the following

```golang
switch strings.ToLower(v1.IPv4Protocol )
```

And yeah, it's easier to type it as lowercase.